src/components/HeaderForm.js

```diff
@@ -165,12 +160,12 @@ class HeaderForm extends Component {
       }
     } catch (error) {
       this.setState({ error: error });
-      console.log(`GET to ${this.props.apiUrl}/getFilenames failed:`, error);
+      console.error(`GET to ${this.props.apiUrl}/getFilenames failed:`, error);
     }
   };
 
   getBedRegions = async (bedFile, dataPath) => {
-    this.setState({ error: null });
+    this.setState((prevState) => ({ ...prevState, error: null }));
```

Comment:
Why does this help at all? Doesn't `setState` implicitly preserve all the keys not mentioned?

Reply:
Reverting this back. I added it out of habit since `useState` requires copying the old properties but you're right, `setState` should preserve keys